src/tools/java/mekanism/tools/common/item/ItemMekanismPaxel.java

```diff
@@ -189,4 +168,15 @@ public void addToBuilder(ImmutableMultimap.Builder<Attribute, AttributeModifier>
         builder.put(Attributes.ATTACK_DAMAGE, new AttributeModifier(BASE_ATTACK_DAMAGE_UUID, "Tool modifier", getAttackDamage(), Operation.ADDITION));
         builder.put(Attributes.ATTACK_SPEED, new AttributeModifier(BASE_ATTACK_SPEED_UUID, "Tool modifier", material.getPaxelAtkSpeed(), Operation.ADDITION));
     }
+
+    // Need to override both method as DiggerItem performs two different behaviors
+    @Override
+    public boolean isCorrectToolForDrops(BlockState state) {
```

Comment:
one of these should just call the other, this solves both cases and removes the duplicate